### PR TITLE
Upgrade to Font awesome v5 + code cleanup

### PR DESCRIPTION
**Pull request by divyanshkul**
Upgrade to font awesome v5 + code cleanup

Minor cleanup

### Submission Checklist

- [x] Your pull request targets the `master` branch of the repository.
- [x] You have only one commit (if not, [squash](http://forum.freecodecamp.org/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [x] You have read the [Contributing guidelines](https://github.com/jboss-outreach/gci/blob/master/CONTRIBUTING.md) and your changes follow them.

#### Type of Change

- [ ] Bug fix
- [x] New implementation

### PR Description

I have made the following changes:
* Upgraded whole website to Font Awesome v5
* Added Gitter chat badge in the footer
* Deleted unused code in ```main.js``` 

Addresses Issue https://github.com/jboss-outreach/gci/issues/112

### CODE REVIEW

**Comment by divyanshkul** [comment]
Hey @aashutoshrathi I have added the giiter icon in this PR. Please see the footer :)

File: _includes/popup.html [hunk 1]
```diff
@@ -22,30 +22,30 @@ <h3> {{ include.person.name }}</h3>
...
+        <i class="fab fa-github" style="font-size: 35px; margin: 10px;"></i>
     </a>
     {% endif %}
 
 
     {% if include.person.facebook %}
     <a href="//facebook.com/{{include.person.facebook}}"
        target="_blank">
-        <i class="fa fa-facebook" style="font-size: 40px; margin: 10px;"></i>
+        <i class="fab fa-facebook-f" style="font-size: 35px; margin: 10px;"></i>
```
Comment (aashutoshrathi) [inline _includes/popup.html]: Why `-f`?

Comment (divyanshkul) [reply _includes/popup.html]: Oh so, right now on our website the facebook icon is just an "f" (not the square icon) Not able to explain exactly, You may look at the differnce  with [fab fa-facebook](https://fontawesome.com/icons/facebook?style=brands) and [fab fa-facebook-f](https://fontawesome.com/icons/facebook-f?style=brands) the syntax for just the "f" icon changed in v5 I think

**Comment by divyanshkul** [comment]
Hey @aashutoshrathi reduced the font size of gitter icon a little (earlier, it was the same size as the other icons in the footer, but it was looking bigger because of its design). Also forgot to add the rss ```fas fa-rss`` syntax to the css, added that as well. 
The last commit was to trigger another build, the bot wasn't working on the earlier one.

**Pull request merged**